src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -195,6 +204,13 @@ create_distributed_table(PG_FUNCTION_ARGS)
 							   colocateWithTableName, ShardCount,
 							   ShardReplicationFactor);
 
+	/* copy over data from regular relations */
+	relationKind = get_rel_relkind(relationId);
+	if (relationKind == RELKIND_RELATION)
+	{
+		CopyLocalData(relationId);
```

Comment:
ConvertToDistributedTable does not create shards, and we can't do anything before the shards are created.